frontend/src/components/suggestions/SuggestionsList.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { FC, useState } from 'react';
+import { Suggestion } from './types';
+import { SuggestionItem } from './SuggestionItem';
+import { Typography, Box, Checkbox, Button, makeStyles, createStyles, Theme } from '@material-ui/core';
+import { Playlist } from '../playlists/types';
+
+const useStyles = makeStyles((theme: Theme) =>
```

Comment:
Should we externalize in separated `.css` file the styles ?

Reply:
Not really since we're using Css-in-js so the purpose is to keep the style close to the component so that it's configurable with the components props